sklearn/covariance/graph_lasso_.py

```diff
@@ -217,10 +217,11 @@ def graph_lasso(emp_cov, alpha, cov_init=None, mode='cd', tol=1e-4,
                 precision_[idx, idx] = (
                     1. / (covariance_[idx, idx]
                           - np.dot(covariance_[indices != idx, idx], coefs)))
-                precision_[indices != idx, idx] = (- precision_[idx, idx]
-                                                   * coefs)
-                precision_[idx, indices != idx] = (- precision_[idx, idx]
-                                                   * coefs)
+                with np.errstate(invalid = 'ignore'):
```

Comment:
> Can someone tell if this is the right way to do it? I was getting these warnings:

Looks good to me, it seems like this technique is used in other tests.

You should write `np.errstate(invalid='ignore')` if you want to be PEP8 compliant though.